azureVMCreateEnvVariables.md

```diff
@@ -0,0 +1,149 @@
+---
```

Comment:
This is the third document that looks the same document. Not sure what is going on here. Are these development files? in which case put them in a dev folder. If not why the duplication?

Reply:
This is the same document which uses 3 different patterns for environment variables/parameters. One uses a .ini input file for variables. One uses a comment block which is hidden to readers, and one uses just plain hardcoded values. I moved them to a demo_scripts folder and added a header at the beginning to clarify the difference.